File: t/VB/OmrProcessingDemo/UI/ResultsPanel/DataGridViewDoubleBuffered.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Public Class DataGridViewDoubleBuffered
   Inherits DataGridView

   Public Sub New()
      MyBase.New()
      DoubleBuffered = True
   End Sub

   Protected Overrides Sub OnScroll(ByVal e As ScrollEventArgs)
      Me.Invalidate()
      MyBase.OnScroll(e)
   End Sub
End Class

File: t/CS/OmrProcessingDemo/UI/ResultsPanel/DataGridViewDoubleBuffered.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace OmrProcessingDemo.UI.ResultsPanel
{
   public class DataGridViewDoubleBuffered : DataGridView
   {
      public DataGridViewDoubleBuffered() : base()
      {
         DoubleBuffered = true;
      }

      protected override void OnScroll(ScrollEventArgs e)
      {
         this.Invalidate();
         base.OnScroll(e);
      }
   }
}
